v0/destinations/drip/data/DripTrack.json

```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "destKey": "action",
+    "sourceKeys": "properties.action",
+    "required": false
+  },
+  {
+    "destKey": "prospect",
+    "sourceKeys": "properties.prospect",
+    "required": false
+  },
+  {
+    "destKey": "occurred_at",
+    "sourceKeys": ["timestamp", "originalTimestamp"],
```

Comment:
lets put a note on this. maybe we should call it `customValues` in the input payload. similar to identify.